nemo/collections/tts/torch/tts_tokenizers.py

```diff
@@ -215,6 +216,8 @@ def __init__(
         """
 
         de_alphabet = "abcdefghijklmnopqrstuvwxyzäöüß"
+        if phonemes == True:
+            de_alphabet = "abcdefghijklmnopqrstuvwxyzäöüßʊʃŋɜːɛɾəɪçɔøɡœɑÜ„1Q̃ɒʒÄɹÖʌθàó̈ðéɐá"
```

Comment:
nit: to avoid human mistakes, it would be better to write it as: `de_alphabet += "ʊʃŋɜːɛɾəɪçɔøɡœɑÜ„1Q̃ɒʒÄɹÖʌθàó̈ðéɐá"` or in a general way, declare a `de_ipa = "ʊʃŋɜːɛɾəɪçɔøɡœɑÜ„1Q̃ɒʒÄɹÖʌθàó̈ðéɐá"` after line 218, and write it as `de_alphabet += de_ipa`